### DISCUSSION TITLE
Concat String Literals with Select statements in Drift files

### DISCUSSION THREAD

**Discussion by muddi900** [simolus3/drift#2704] [Q&A]
I am trying to do something like this

```sql
INSERT INTO table1 (field1)
VALUES('Lorem' || (SELECT CAST(int_field AS TEXT) Limit 1) || 'Ipsum')
```

as part of a Trigger. However the trigger does not fire. I do not see an error message. How do I achieve this.

**Comment by simolus3** [comment]
The problem is probably not related to the the trigger's body. Did you write a migration for the trigger to add it to the database schema after defining it?
